### PR TITLE
Some code clean-up and refactoring

### PR DESCRIPTION
**Pull request by Haplois**
Moved the code clean-up and refactoring done in #794 to here.

### CODE REVIEW

File: test/E2ETests/Automation.CLI/Automation.CLI.csproj [hunk 1]
```diff
@@ -33,7 +33,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.TestPlatform.AdapterUtilities, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.TestPlatform.AdapterUtilities.$(TestPlatformVersion)\lib\net45\Microsoft.TestPlatform.AdapterUtilities.dll</HintPath>
```
Comment (nohwnd) [inline test/E2ETests/Automation.CLI/Automation.CLI.csproj:36]: Is this intended? #Resolved

File: src/Adapter/PlatformServices.Interface/ObjectModel/ITestMethod.cs [hunk 2]
```diff
@@ -46,7 +47,7 @@ public interface ITestMethod
         /// Gets the fully specified method name metadata format.
         /// </summary>
         /// <example>
-        ///     <code>MethodName`2(ParamTypeA,ParamTypeB,�)</code>
+        ///     <code>MethodName`2(ParamTypeA,ParamTypeB,�)</code>
```
Comment (nohwnd) [inline src/Adapter/PlatformServices.Interface/ObjectModel/ITestMethod.cs]: Broken encoding? #Resolved

Comment (Haplois) [reply src/Adapter/PlatformServices.Interface/ObjectModel/ITestMethod.cs]: Every time I save that file, VS Code breaks the encoding... Thanks for pointing it out. #Resolved

File: test/E2ETests/Automation.CLI/Automation.CLI.csproj (hunk 1 above)
Comment (Haplois) [reply test/E2ETests/Automation.CLI/Automation.CLI.csproj:36]: Yes, replaced with `netstandard2.0` version. #Resolved

**Review by cvpoienaru — APPROVED** [review]
LGTM

**Pull request merged**